inferlo/pairwise/optimization/convex_bounds.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+import cvxpy as cp
+
+if TYPE_CHECKING:
+    from inferlo import PairWiseFiniteModel
+
+from inferlo.pairwise.testing import grid_potts_model, tree_potts_model, \
+    line_potts_model
+from inferlo.pairwise.optimization.path_dp import max_lh_path_dp
+
+def LPrelaxation(model: PairWiseFiniteModel):# -> np.ndarray:
```

Comment:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.
python.org/dev/peps/pep-0008/#function-and-variable-names